cluster_experiments/power_config.py

```diff
@@ -122,6 +126,48 @@ class PowerConfig:
 
     seed: Optional[int] = None
 
+    def __post_init__(self):
+        if "switchback" not in self.splitter:
+            if self._are_different(self.switch_frequency, None):
+                self._set_and_log("swithc_frequency", None, "splitter")
```

Comment:
```suggestion
                self._set_and_log("switch_frequency", None, "splitter")
```

Reply:
I got this and other minor errors thanks to the new tests added  🚀